### PR TITLE
theme-rebellion - A suitably dark theme Rev #2

### PR DESCRIPTION
**Pull request by marjohn56**
There are four files that cause issues with this theme, they are outside of the themes folder and will require modifications.

The first two

gateways.widget.php
status_gateways.php

Have fixed colours and a couple of them are just wrong for this theme.

The other two

jquery.tokenize.css - Affects the the java handled pages such Intrusion protection

nv.d3.css - Affects nv.d3 objects and the text is black rather than what it needs to be.

### CODE REVIEW

File: misc/theme-rebellion/Makefile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+PLUGIN_NAME=		theme-rebellion
+PLUGIN_VERSION=		0.1
+PLUGIN_COMMENT=		A suitably dark theme
+PLUGIN_MAINTAINER=	team-rebellion@queena-park.com
```
Comment (fichtner) [inline misc/theme-rebellion/Makefile]: typo in domain?

Comment (marjohn56) [reply misc/theme-rebellion/Makefile]: Oh jeez!

Comment (fichtner) [reply misc/theme-rebellion/Makefile]: almost there, don't give up :)

**Comment by marjohn56** [comment]
Corrected... :)

Now I need to create the email forward for that account. :)

**Comment by fichtner** [comment]
Merged, thank you. ❤️

**Pull request merged**